.github/workflows/main_uicrafter.yml

```diff
@@ -0,0 +1,65 @@
+# Docs for the Azure Web Apps Deploy action: https://github.com/Azure/webapps-deploy
+# More GitHub Actions for Azure: https://github.com/Azure/actions
+
+name: Build and deploy ASP.Net Core app to Azure Web App - UICrafter
+
+on:
+  push:
+    branches:
+      - main
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest

```

Comment:
The apk runs on windows-2022. Is that important?

Reply:
I think latest is fine but it needs to run on windows and not linux, Ill change to latest